contracts/contracts.go

```diff
@@ -0,0 +1,145 @@
+/*
+Package contracts embeds compiled Neo contracts and provides access to them.
+*/
+package contracts
+
+import (
+	"embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/nef"
+)
+
+// Contract groups information about Neo contract stored in the current package.
+type Contract struct {
+	NEF      nef.File
+	Manifest manifest.Manifest
+}
+
+//go:embed *.nef *.manifest.json
+var _fs embed.FS
+
+// Read reads compiled contracts stored in the package sorted numerically.
+// File schema:
+//   - compiled executables (NEF) are named by pattern 'N.nef'
```

Comment:
But why not `N-name.nef`?

Reply:
i don't mind to add names, required or optional (allow `N.nef` for any custom contract)?

Reply:
Either way, I'd go with required just to be consistent and have a bit more visibility into the mix.

Reply:
done, require `a-z` name for now